lib/ansible/modules/cloud/amazon/dynamodb_table.py

```diff
@@ -411,9 +462,20 @@ def main():
     except (NoAuthHandlerFound, AnsibleAWSError) as e:
         module.fail_json(msg=str(e))
 
+    if module.params.get('tags'):
+        try:
+            region, ec2_url, aws_connect_kwargs = get_aws_connection_info(module, boto3=True)
+            boto3_dynamodb = boto3_conn(module, conn_type='client', resource='dynamodb', region=region, endpoint=ec2_url, **aws_connect_kwargs)
+            boto3_sts = boto3_conn(module, conn_type='client', resource='sts', region=region, endpoint=ec2_url, **aws_connect_kwargs)
```

Comment:
Switching the whole module over to boto3 would require the users of the module to add boto3 1.4.4 as a dependency. I chose to implement it this was for backwards compatibility reasons. I agree that it would be great to change the module over to exclusively use boto3 1.4.4+, but I think this is the best way to allow a transition period for the users. I think the switch to boto3 1.4.4+ should happen in a future Ansible release and perhaps a different module altogether. That said, I don't really know how dependencies are managed between Ansible versions and the deprecation policy.